.github/workflows/test-mitosheet-frontend.yml

```diff
@@ -119,16 +121,30 @@ jobs:
       matrix:
         os: ['ubuntu-latest', 'macos-latest', 'windows-latest']
         python-version: ['3.10']
-        project: ['chromium', 'firefox', 'webkit', 'Google Chrome', 'Microsoft Edge']
+        project: 
+          - test-name: 'chromium'
+            install-name: 'chromium'
+          - test-name: 'firefox'
+            install-name: 'firefox'
+          - test-name: 'webkit'
+            install-name: 'webkit'
+          - test-name: 'Google Chrome'
+            install-name: 'chrome'
+        testfiles: ['context_menus', 'features', 'toolbar']
```

Comment:
Right now there are tests that would not get run as a result of this -- `set_column_formula_tests.spec.ts` because its not in the features folder (my bad on that!)

It seems like the vast majority of tests are going to be in the `features` test, so maybe we will eventually want to change the breakdown, but looks fine for now.

Reply:
Agreed